src/index.json

```diff
@@ -56400,6 +56400,51 @@
                 "sha256Digest": "58074ff73a72a1a9c76d406a4b6988a7b714dfaa9259b92172a7b6f66680db31"
             }
         ],
+        "stack-hci-vm": [
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/SelfServiceVM/CLI/stack_hci_vm-0.1.2-py3-none-any.whl",
+                "filename": "stack_hci_vm-0.1.2-py3-none-any.whl",
+                "metadata": {
+                    "azext.isExperimental": true,
+                    "azext.minCliCoreVersion": "2.15.0",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
+                                    "name": "Microsoft Corporation",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-cli-extensions/tree/master/src/stack-hci-vm"
+                            }
```

Comment:
As the source code of this extension is not in Azure/azure-cli-extensions repo, please correct the descriptions here.

Reply:
What all do I need to correct?

Reply:
Is it okay if we remove the whole section?
@kairu-ms

Reply:
These metadata are retrieved from the extension source code. So you should correct them in the extension code.